core/src/main/scala/kafka/log/LogCleanerManager.scala

```diff
@@ -64,6 +64,19 @@ private[log] class LogCleanerManager(val logDirs: Array[File], val logs: Pool[To
   @volatile private var dirtiestLogCleanableRatio = 0.0
   newGauge("max-dirty-percent", new Gauge[Int] { def value = (100 * dirtiestLogCleanableRatio).toInt })
 
+  /* a gauge for tracking the number of logCleaner runs */
+  @volatile private var numLogCleanRuns = 0
+  newGauge("num-runs", new Gauge[Int] { def value = numLogCleanRuns })
+
+  /* a gauge for tracking the last time of logCleaner run */
+  @volatile private var logCleanLastRun : Long = System.currentTimeMillis
+  newGauge("last-run-time", new Gauge[Long] { def value = logCleanLastRun })
+
+  /* a gauge for tracking the number of filthy logs. A non zero value indicates
```

Comment:
@wushujames @kiranptivo so from this discussion it seems like this metric is really for "are the log cleaner threads keeping up", right? We have a better metric for "is the thread completely dead" (time since last run, above). Is there a better metric for "is the log cleaner thread keeping up" than `num-filthy-logs` that is easier to understand from the user's perspective? I generally like exposing percentages or other things that are a relatively static threshold that don't need to be tuned for every single use case on a one-off basis.